apis/r/DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Imports:
     Matrix,
     stats,
     bit64 (>= 4.6.0.1),
-    tiledb (>= 0.30.0),
+    tiledb (>= 0.31.1),
```

Comment:
@jp-dark I've never ever done this for core-rc dependency PRs. (Please see https://github.com/single-cell-data/TileDB-SOMA/pulls?q=is%3Apr+is%3Aclosed.) And it's not in our [established released procedure](https://github.com/single-cell-data/TileDB-SOMA/wiki/Branches-and-releases) (maybe it should be!).

If you want to do this, and if it works, great! cc @mojaveazure for 👀 though.

Reply:
I'm all for updating the minimum R version, though in this case it would be `>= 0.32.0` (or 0.31.1.1 for the development version) as that will be the next version of tiledb-r (currently unreleased, but I can cut a new tiledb-r once core 2.28 is officially out)

I will point out that pinning on a version of tiledb-r that falls back on core 2.28 will require holding off on a CRAN submission of tiledbsoma-r until that new version of tiledb-r is on CRAN (should be fast, no manual reviews needed there)